workers/unity/Packages/com.improbable.gdk.tools/DevAuthTokenUtils.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Improbable.Gdk.Tools.MiniJSON;
+using UnityEditor;
+using UnityEngine;
+
+namespace Improbable.Gdk.Tools
+{
+    public static class DevAuthTokenUtils
+    {
+        private static readonly string JsonDataKey = "json_data";
+        private static readonly string TokenSecretKey = "token_secret";
+
+        [MenuItem("SpatialOS/Generate Dev Authentication Token", false, MenuPriorities.GenerateDevAuthToken)]
+        private static void Generate()
+        {
+            var devAuthToken = string.Empty;
+            var gdkToolsConfiguration = GdkToolsConfiguration.GetOrCreateInstance();
+            var devAuthTokenDir = gdkToolsConfiguration.DevAuthTokenFullDir;
+            var devAuthTokenFilePath = gdkToolsConfiguration.DevAuthTokenFilepath;
+            var devAuthTokenLifetimeHours = $"{gdkToolsConfiguration.DevAuthTokenLifetimeHours}h";
+
+            var receivedMessage = string.Empty;
+            RedirectedProcess
+                .Command(Common.SpatialBinary)
+                .WithArgs("project", "auth", "dev-auth-token", "create", "--description", "\"Dev Auth Token\"",
+                    "--lifetime", devAuthTokenLifetimeHours, "--json_output")
+                .InDirectory(Common.SpatialProjectRootDir)
+                .AddOutputProcessing(message => receivedMessage = message)
```

Comment:
Are we guaranteed to only have exactly one line of output forever and ever, or that the last line received is always the content that we want?

Reply:
there can definitely be multiple lines (e.g. spatial telling us to update), but the last one needs to be the json output